samples/client/petstore/python/swagger_client/apis/fake_api.py

```diff
@@ -103,6 +104,31 @@ def test_endpoint_parameters(self, number, double, string, byte, **kwargs):
         if ('byte' not in params) or (params['byte'] is None):
             raise ValueError("Missing the required parameter `byte` when calling `test_endpoint_parameters`")
 
+        if 'number' in params and params['number'] > 543.2: 
+            raise ValueError("Invalid value for parameter `number` when calling `test_endpoint_parameters`, must be a value less than or equal to  `543.2`")
+        if 'number' in params and params['number'] < 32.1: 
+            raise ValueError("Invalid value for parameter `number` when calling `test_endpoint_parameters`, must be a value greater than or equal to `32.1`")
+        if 'double' in params and params['double'] > 123.4: 
+            raise ValueError("Invalid value for parameter `double` when calling `test_endpoint_parameters`, must be a value less than or equal to  `123.4`")
+        if 'double' in params and params['double'] < 67.8: 
+            raise ValueError("Invalid value for parameter `double` when calling `test_endpoint_parameters`, must be a value greater than or equal to `67.8`")
+        if 'string' in params and not re.search('[a-z]', params['string'], flags=re.IGNORECASE): 
+            raise ValueError("Invalid value for parameter `string` when calling `test_endpoint_parameters`, must conform to the pattern `/[a-z]/i`")
```

Comment:
When I looked this earlier (http://stackoverflow.com/a/23613042/677735), I was thinking about updating the `pattern` to `'FIle:/+(.*)',re.IGNORECASE` with the following template:
```
{{#pattern}}
pattern = re.compile({{{.}}})
if pattern.match(params['{{name}}]):
...
{{/pattern}}
```
On course your solution using vendor extension works as well.